src/components/ec-checkbox/ec-checkbox.vue

```diff
@@ -0,0 +1,212 @@
+<template>
+  <div class="ec-checkbox">
+    <input
+      :id="id"
+      v-model="inputModel"
+      :aria-describedby="errorId"
+      type="checkbox"
+      class="ec-checkbox__input"
+      :disabled="disabled"
+    >
+    <label
+      :for="id"
+      class="ec-checkbox__label"
+      :class="{
+        'ec-checkbox__label--has-error': isInvalid || $slots['error-message'],
```

Comment:
You are checking two times the **$slots['error-message']**

Reply:
On the "ec-checkbox" unlike "ec-input-field" we decided that should be able to pass the error and label messages from both from props and slots.

Because the error can come from 2 different places I am checking if one those places is true so I can add the class.